README.md

```diff
@@ -1,3 +1,50 @@
 Ray-based Apache Beam Runner
 ==========================
-This is a WIP proof of concept implementation undergoing frequent breaking changes and should not be used in production.
\ No newline at end of file
+This is a WIP proof of concept implementation undergoing frequent breaking changes and should not be used in production.
+
+
+## The Portable Ray Beam Runner
+
+The directory `ray_beam_runner/portability` contains a prototype for an implementation of a [Beam](https://beam.apache.org)
+runner for [Ray](https://ray.io) that relies on Beam's portability framework.
+
+### Install and build from source
+
+To install the existing Ray Beam runner from a clone of this repository, you can follow the next steps:
+
+```shell
+# First create a virtual environment to install and run Python dependencies
+virtualenv venv
+. venv/bin/activate
+
+# Install development dependencies for the project
+pip install -r requirements_dev.txt
```

Comment:
Is this step still needed, or should everything be in the test dependencies already?

Reply:
we still need this to install the autoformaters and checkers (E.g. black, flake8)

Reply:
Ah, yes, of course :)